apps/gitness/src/pages/execution/execution-details.tsx

```diff
@@ -9,14 +9,30 @@ import {
   StageExecution,
   ContactCard,
   convertExecutionToTree,
-  StageProps
+  StageProps,
+  getStepId,
+  parseStageStepId
 } from '@harnessio/playground'
 import { PathParams } from '../../RouteDefinitions'
 import { useGetRepoRef } from '../../framework/hooks/useGetRepoPath'
 import { ExecutionState } from '../../types'
-import { getDuration, timeAgoFromEpochTime } from '../pipeline-edit/utils/time-utils'
+import { getDuration, timeAgoFromEpochTime, formatDuration } from '../pipeline-edit/utils/time-utils'
+import useSpaceSSE from '../../framework/hooks/useSpaceSSE'
+import { useGetSpaceURLParam } from '../../framework/hooks/useGetSpaceParam'
+
+/**
+ * Ensure this should come from @harnessio/code-service-client instead
+ */
+enum ExecutionEvent {
```

Comment:
should we move this enum to types.ts or interface.ts file

Reply:
Done, moved to type.ts under `apps/gitness/src`.